Prog/2019/Internet Access.sas

```diff
@@ -0,0 +1,183 @@
+      hshldinc50000to74999_2013_17+ hshldinc75000to99999_2013_17 ;
+      percent100Kplus= hshldinc100000plus_2013_17/tothhdemom;
+	  percentunder15K= hshldincunder15000_2013_17/tothhdemom;
+	  percent15to50K= (hshldinc15000to34999_2013_17+ hshldinc35000to49999_2013_17)/tothhdemom;
+	  percent50to75K= hshldinc50000to74999_2013_17/tothhdemom;
+      percent75Kto100K= hshldinc75000to99999_2013_17/tothhdemom;
+      internetdenom= Numdialup_&year. + Numbroadbandall_&year. + Numcellular_&year. + Numcellularonly_&year. + Numbroadband_&year. + Numbroadbandonly_&year. + Numsatellite_&year.+ 
+           Numsatelliteonly_&year. + Numotheronly_&year. + Numaccesswosub_&year. + Numnointernet_&year.;
+	  pctinternet= (internetdenom- Numnointernet_&year.)/internetdenom;
+	  pctnointernet= 1- pctinternet;
+
+
+	  format Jurisdiction Jurisdiction. ;
+run;
+
+proc export data= BroadbandAcccess 
+   outfile="&_dcdata_default_path\Requests\Prog\2019\Broadband_access_tract.csv"
+   dbms=csv
+   replace;
+run;
+
+proc sort data= BroadbandAcccess ;
+by metro15;run;
+
+proc summary data = BroadbandAcccess;
+	var  totpop_&year. medfamincm_&year. popalonew_&year. popaloneb_&year. popaloneh_&year. popalonea_&year. numpopbroadbandtot
+	       numpopbroadbanda_&year. numpopbroadbandiom_&year. numpopbroadbandb_&year. numpopbroadbandh_&year. numpopbroadbandw_&year.
+	       hshldinc100000plus_2013_17 hshldincunder15000_2013_17 hshldinc15000to34999_2013_17 hshldinc35000to49999_2013_17
+           hshldinc50000to74999_2013_17 hshldinc75000to99999_2013_17;
+		   by metro15;
+	output out = AccessinDMV sum = ;
+run;
+proc sort data=InternetAcccess;
+by Jurisdiction;run;
+proc summary data = InternetAcccess (where=(metro15= "47900"));
+	var  totpop_&year. medfamincm_&year. popalonew_&year. popaloneb_&year. popaloneh_&year. popalonea_&year. numpopbroadbandtot
+	       numpopbroadbanda_&year. numpopbroadbandiom_&year. numpopbroadbandb_&year. numpopbroadbandh_&year. numpopbroadbandw_&year.
+	       hshldinc100000plus_2013_17 hshldincunder15000_2013_17 hshldinc15000to34999_2013_17 hshldinc35000to49999_2013_17
+           hshldinc50000to74999_2013_17 hshldinc75000to99999_2013_17;
+		   by Jurisdiction;
+	output out = Accessbycounty sum = ;
+run;
+data accessbycounty2;
+set Accessbycounty;
+tothhdemom= hshldinc100000plus_2013_17+ hshldincunder15000_2013_17+ hshldinc15000to34999_2013_17+ hshldinc35000to49999_2013_17+
+           hshldinc50000to74999_2013_17+ hshldinc75000to99999_2013_17 ;
+      percent100Kplus= hshldinc100000plus_2013_17/tothhdemom;
+	  percentunder15K= hshldincunder15000_2013_17/tothhdemom;
+	  percent15to50K= (hshldinc15000to34999_2013_17+ hshldinc35000to49999_2013_17)/tothhdemom;
+	  percent50to75K= hshldinc50000to74999_2013_17/tothhdemom;
+      percent75Kto100K= hshldinc75000to99999_2013_17/tothhdemom;
+	  broadbandaccess= numpopbroadbandtot/totpop_&year.;
```

Comment:
not the correct denominator for broadband access. and numpopbroadbandtot is not created anywhere.